src/main/java/com/Feature1/WSFeaure1/UserController.java

```diff
@@ -0,0 +1,17 @@
+package com.Feature1.WSFeaure1;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+
+@RestController
+public class UserController {
+    private AtomicInteger countr = new AtomicInteger();
+    @RequestMapping("/createUser")
+     public User createUser (@RequestParam(value = "name", defaultValue = "???") String name) {
```

Comment:
@Xeniia-droid You don't have to provide the `defaultValue` unless you want to 

UPD: I won't review this class for now because it seems to be work in progress. Ping me if you have any questions.